tests/Unit/UseCase/BannerSelection/BannerSelectionUseCaseTest.php

```diff
@@ -55,4 +56,32 @@ public function test_given_one_percent_ratio_then_limit_is_applied_for_two_perce
 		$this->assertEquals( null, $bannerSelectionData->getVisitorData()->getBucketIdentifier() );
 		$this->assertEquals( 0, $bannerSelectionData->getVisitorData()->getTotalImpressionCount() );
 	}
+
+	public function test_given_banner_is_returned_then_view_count_is_incremented() {
```

Comment:
I think for these test cases, `when` makes more sense than `given`, because `banner_is_returned` describes the scenario and not the precondition (see https://martinfowler.com/bliki/GivenWhenThen.html).